src/angularjs/src/app/places/list/places-list.controller.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @ngdoc controller
+ * @name pfb.place.list.controller:PlaceListController
+ *
+ * @description
+ * Controller for listing analysis jobs
+ *
+ */
+(function() {
+    'use strict';
+
+    /** @ngInject */
+    function PlaceListController($state, $stateParams, $scope, Pagination, AuthService,
+                                 Neighborhood, AnalysisJob) {
+        var ctl = this;
+
+        var defaultParams = {
+            limit: null,
+            offset: null,
+            latest: 'True'
+        };
+        var nextParams = {};
+        var prevParams = {};
+
+        initialize();
+
+        function initialize() {
+            ctl.isAdminUser = AuthService.isAdminUser();
+
+            ctl.hasNext = false;
+            ctl.getNext = getNext;
+
+            ctl.hasPrev = false;
+            ctl.getPrev = getPrev;
+            ctl.places = [];
+
+            ctl.filters = {};
+
+            getPlaces();
+        }
+
+        function getPlaces(params) {
+            params = params || $stateParams;
```

Comment:
Good catch. Yes, `$stateParams` is used elsewhere for passing around ID filters and such. Useless for the list here at the moment, but maybe it might come in handy.